website_sale_wishlist/data/ir_cron.yml

```diff
@@ -0,0 +1,9 @@
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+- !record {id: garbage_collector, model: ir.cron}:
```

Comment:
This is for removing wishlist items for non-users and sessions that no longer exist